ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -1092,8 +1092,8 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-private-workers-marketimage-p2-f14
-  cron: 18 7 13,28 * *
+- as: baremetal-azure-ipi-private-workers-marketimage-p2-f28
```

Comment:
Adding `disabled` to the ignore keyword in https://github.com/openshift/release/pull/40993
So that instead of using prefix baremetal for azure profiles, we can use `disabled` as the prefix.